### DISCUSSION TITLE
Clear backpack button

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#2859] [Ideas]
I have a ton of things that I don't even use but it will take hours to clean my backpack.
This could be a nice feature if you ask me.

**Comment by CloudGate1234** [comment]
this is good idea

**Comment by Endermaster3690** [comment]
we'll want to add a dangerous label to it and add a confirmation.

**Comment by TheColaber** [comment]
This was also suggested by griffpatch 👀 and also @Endermaster3690 a confirmation would just be ok. No need for the label if we do that.
